### PR TITLE
Removed the PR table example (this is now included by GitHub template)

### LINKED ISSUE #6357: Update the PR table in the contributing guide
As pointed by @vladapopster, the contributing guide (http://symfony.com/doc/current/contributing/code/patches.html#make-a-pull-request) contains a PR table which is not updated to the latest one used in the Symfony repository (it lacks the `Branch` row for example).

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #6357

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1:

File: contributing/code/patches.rst [hunk 1]
```diff
@@ -268,49 +268,10 @@ pull request message, like in:
...
-.. code-block:: text
-
-    | Q             | A
-    | ------------- | ---
-    | Fixed tickets | [comma separated list of tickets fixed by the PR]
-    | License       | MIT
+The default pull request description contains a table which you must fill in
+with the appropriate answers. This ensures that contributions may be reviewed
+without needless feedback loops and that your contributions can be included into
+Symfony as quickly as possible:
```
Comment (stof) [inline contributing/code/patches.rst]: no need to end the sentence with a colon.

**Comment by weaverryan** [comment]
Good catch! I'll remove that extra colon :)

**Comment by mvar** [comment]
@javiereguiluz Contributing document still links to the PR table example.

**Pull request merged**